lib/sqeduler/worker/synchronization.rb

```diff
@@ -45,11 +53,7 @@ def perform(*args)
       private
 
       def sync_lock_key(*args)
-        if args.empty?
-          self.class.name
-        else
-          "#{self.class.name}-#{args.join}"
-        end
+        self.class.sync_lock_key(*args)
```

Comment:
Usually I prefer to keep things around in case someone did something strange, but that's fair. I removed it.